flowauth/Pipfile

```diff
@@ -14,7 +14,7 @@ zxcvbn = "*"
 versioneer = "*"
 passlib = "*"
 argon2_cffi = "*"
-"psycopg2-binary" = "*"
+psycopg2 = "*"
```

Comment:
Why are we switching back to the non-binary version of `psycopg2`? Is this because we need to build it from source because there is no wheel that matches the alpine distribution?